main.go

```diff
@@ -263,7 +264,7 @@ func registerResources() {
 	cMessageSettings.SetRegistrar(regInstance)                             //Registering conversations messaging settings
 	routingSkillGroup.SetRegistrar(regInstance)                            //Registering routing skill group
 	cMessageSettingsDefault.SetRegistrar(regInstance)                      //Registering conversations messaging settings default
-	location.SetRegistrar(regInstance) //Registering location
+	locationResource.SetRegistrar(regInstance)                             //Registering location
```

Comment:
Apologies, I wasn't clear enough. I meant change the variable to `locationResource`. The package name should always stay as `location`